src/pages/workspaces/workspace/Tools.js

```diff
@@ -25,9 +25,23 @@ const styles = {
     display: 'flex', flexWrap: 'wrap',
     marginRight: listView ? undefined : '-1rem'
   }),
+  // Card's position: relative and the outer/inner styles are a little hack to fake nested links
+  // https://stackoverflow.com/questions/13052598/creating-anchor-tag-inside-anchor-tag
+  card: {
+    ...Style.elements.card, position: 'relative'
+  },
+  outerLink: {
+    position: 'absolute', top: 0, right: 0, bottom: 0, left: 0
+  },
+  innerContent: {
+    position: 'relative', pointerEvents: 'none', zIndex: 1
+  },
+  innerLink: {
+    pointerEvents: 'all'
```

Comment:
nit: should probably be 'auto', since 'all' is technically svg-only (even though it seems to work).